src/audio/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ if(NOT BUILD_LIBRARY)
 			detect_test.c
 		)
```

Comment:
@tlauda please start the commit message with a brief description of what is "PCM converter" and what is its purpose.

Reply:
@dbaluta But it's there: This module provides static functions, which allows to copy
buffer data with frame format conversion.

Reply:
You are right.